README.md

```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
 
 ### Android
 
-Run Mobidex in an Android emulator:
+Start develop in Android:
 
-1.  In the terminal, execute `npm start`.
-2.  Open android studio and import mobidex from the `android` directory
-3.  Comment out `android.enableAapt2=false` in `gradle.properties`
-4.  Sync gradle files
-5.  Click play!
+ 0. The first steps is your workstation is installed dependencies based in [react-native documentation](https://facebook.github.io/react-native/docs/getting-started) or follow the [this steps](#setup).
+ 1. install dependencies of project using yarn with command `yarn` or npm `npm install`.
+ 2. Connect the smartphone or start the your favorite emulator.
+ 3. Run the comand `react-native run-android` to compile and run the install project in android.
+ 4. Run the command `sudo rm -r node_modules/web3-contracts-loader/node_modules/web3`
```

Comment:
Why is this needed?

Reply:
What error are you getting? That shouldn't matter.

Reply:
I have much conflict with different versions of web3 in web-contracts-loader and like the dependency of Mobidex project

Reply:
We can fix this in a separate issue. Please create one.